具体时间:[2021-11-12 00:24:47,515][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 00:24:49,477][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 00:24:49,703][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 00:36:24,344][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 00:36:25,868][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 00:36:26,058][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 00:36:35,314][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 00:52:11,023][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 00:52:12,501][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 00:52:12,675][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 01:37:49,687][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 01:37:51,119][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 01:37:51,311][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 01:37:58,375][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 02:06:01,165][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 02:06:02,756][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:06:03,002][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:06:11,644][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 02:20:28,653][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 02:20:30,258][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:20:30,498][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:20:36,970][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 02:31:36,726][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 02:31:38,380][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:31:38,582][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:31:45,616][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 02:45:52,933][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 02:45:54,583][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:45:54,853][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 02:46:01,766][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 03:11:45,384][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 03:11:46,891][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:11:47,154][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:11:53,535][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 03:13:15,181][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 03:13:16,743][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:13:17,017][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:13:26,837][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 03:40:12,897][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 03:40:14,344][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:40:14,563][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:40:21,483][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 03:41:23,437][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 03:41:24,898][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:41:25,131][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 03:41:31,348][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 04:41:24,598][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 04:43:36,985][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 05:41:25,774][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 06:41:26,948][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 07:41:28,140][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 08:41:29,328][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 09:41:30,505][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 10:41:31,662][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 11:41:32,829][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 12:41:33,975][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 13:41:35,167][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 14:41:36,320][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 15:41:37,467][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 16:41:38,641][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 17:41:39,819][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 18:41:40,989][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 18:44:01,603][default:INFO(182)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/TY_RMS_Multiple_Manage/views.py]模块名:[views:]函数名:[account_login]:
[第182行]-登录成功

具体时间:[2021-11-12 19:41:42,329][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 19:44:29,923][default:INFO(22)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/user/login_required.py]模块名:[login_required:]函数名:[process_request]:
[第22行]-用户没有登录, 跳转到登录页面

具体时间:[2021-11-12 20:41:43,543][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


具体时间:[2021-11-12 21:41:44,691][default:DEBUG(223)]
当前文件位置:[/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py]模块名:[SupervisionWarning:]函数名:[supervision_start_thread]:
[第223行]-预警监测：Traceback (most recent call last):
  File "/home/yanyi/Project/MainManage/ServerData/Lib/SupervisionWarning.py", line 49, in supervision_start_thread
    if data_obj.InventoryWarningValue > data_obj.normal_count:
TypeError: '>' not supported between instances of 'str' and 'decimal.Decimal'


